checker/checker_test.go

```diff
@@ -1129,7 +1129,7 @@ func TestCheck(t *testing.T) {
 		t.Run(name, func(tt *testing.T) {
 			// Runs the tests in parallel to ensure that there are no data races
 			// due to shared mutable state across tests.
-			tt.Parallel()
+			// tt.Parallel()
```

Comment:
this is on purpose?

Reply:
Sorry, likely something I did during debug. The pull is still a work in progress (comments, code hygiene), but works correctly. I'm going to go over the changes with a fine tooth comb before I request review.